src/constants.ts

```diff
@@ -91,6 +95,12 @@ export const LogTypes: Record<LogType, Partial<LogObject>> = {
   log: {
     level: LogLevels.log,
   },
+  group: {
```

Comment:
group/groupEnv don't need to be a log type (like consola.prompt they can be standalone consola methods)

Reply:
Treating them as log types simplifies integration with the existing logic, I suppose 🤔

- `wrap` & `unwrap`
- reporters can format or process `.group` message with the same `logObj`